### ISSUE TITLE
Avoid compilation on whitespace changes

### ISSUE THREAD

**Issue by faassen** [wasp-lang/wasp#437]
When I do whitespace changes the recompilation procedure takes place. Would it be possible to detect that the essential configuration is still the same and avoid recompilation upon save?

**Comment by Martinsos** [comment]
Good point! I believe we could do that by running only compiler frontend, which would build AppSpec (or maybe stop even sooner, after tokenization), and then we could compare the result with the result of latest compilation, which we would keep stored somewhere (probably on the disk). We could compare checksums instead of actual source. We would need to extend this in the future to also work correctly for multiple .wasp files once we add support for them.

So we could certainly do it, however it would probably make sense to postpone this optimization for quite some later, maybe when Wasp is stable (1.0).
